source/read-write-concern/tests/README.rst

```diff
@@ -52,6 +52,26 @@ array of test case objects, each of which have the following keys:
 - ``isServerDefault:`` Indicates whether the read or write concern is considered the server's default.
 - ``isAcknowledged:`` Indicates if the write concern should be considered acknowledged.
 
+Operation
+~~~~~~~~~
+
+These tests check that the default write concern is omitted in operations.
+
+The spec test format is an extension of `transactions spec tests <https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst>`_ with the following additions:
+
+- ``clientOptions`` may include a ``writeConcern``, distinct from the URI option ``w``, to indicate a client level write concern.
```

Comment:
I think this may cause more trouble than its worth, since most test runners currently expect `clientOptions` to contain known URI options. Related to my earlier comment about possible redundancy specifying client, database, _and_ collection options can we just use database _or_ collection options depending on the object used to execute the operation? I think `runCommand` would be the only thing requiring database options in that case.

Reply:
Removed, and simplified.